integration/docker/network_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package docker
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("network", func() {
+	var (
+		args []string
+	)
+
+	AfterEach(func() {
+		args = []string{"network", "rm", "my-bridge-network"}
+		runDockerCommand(0, args...)
+		args = []string{"network", "ls"}
+		stdout := runDockerCommand(0, args...)
+		Expect(stdout).NotTo(ContainSubstring("my-bridge-network"))
```

Comment:
Nit: there are 4 occurrences of this string, so I'd be tempted to use a variable to make maintenance easier.